management-web/src/main/webapp/WEB-INF/pages/product.jsp

```diff
@@ -0,0 +1,22 @@
+<%@include file="../commons/header.jsp" %>
```

Comment:
I would recommend renaming this file as addProductForm or something more descriptive, product gives the idea is for showing a product